### ISSUE TITLE
CG-GUI creating toml files outside of spec

### ISSUE THREAD

**Issue by Luke2Sky** [CrystalGrowerOrg/cgaspects#97]
I don't know if this is the right place as it pertains to CrystalGrower itself, but is not totally related to CGaspects (I saw issues that pertained to CrystalGrower so I decided to create this one). But, the CG-OCC gui is creating out of spec toml files. Firstly, booleans have to be uncapitalised. Also, paths and strings have to be in quotation marks. I realized this after having problems using the Python toml library to read and change various toml files so I can run a lot of calculations for different polymorphs overnight. OCC seems to run fine even though the toml is out of spec, but I thought it would be good to create this issue. The nice thing about toml is that it should map to any language's hash table, in Python, being a dictionary. The spec in question is the following: https://toml.io/en/

**Comment by peterspackman** [comment]
You're right that these are not valid toml files. The issue is largely that the occ running bit of code was hastily cobbled together, and occ doesn't really require the toml standard, but whatever CLI11 will parse (which is also `ini` files).

I probably shouldn't have given it the toml suffix without it being valid toml, I'll try and fix that in future versions.
